lib/http-outgoing.js

```diff
@@ -146,6 +150,24 @@ export default class PodletClientHttpOutgoing extends PassThrough {
         this.#redirectable = redirectable;
     }
 
+    get js() {
+        // return the internal js value or, fallback to the manifest for backwards compatibility
+        return this.#js || this.#manifest.js;
+    }
+
+    set js(value) {
+        this.#js = value;
+    }
+
+    get css() {
+        // return the internal css value or, fallback to the manifest for backwards compatibility
+        return this.#css || this.#manifest.css;
+    }
+
+    set css(value) {
+        this.#css = value;
+    }
+
```

Comment:
These are js and css getter/setters that are used whenever the content route is fetched OR when pushFallback is called (at which point the cached js and css are set)